packages/ra-core/src/types.ts

```diff
@@ -67,9 +67,14 @@ export type AuthProvider = {
     checkError: (error: any) => Promise<void>;
     getIdentity?: () => Promise<UserIdentity>;
     getPermissions: (params: any) => Promise<any>;
+    handleLoginCallback?: <
```

Comment:
I find the name too long. Possible alternatives:

- `handleCallback`
- `afterLogin`
- `processToken`

I think we should emphasize the "token" in the usage doc - the only utility of the redirection is actually to pass the token, right?

Reply:
As dicussed there might be more than just a token. I think `handleCallback` is probably the best